openprescribing/templates/base.html

```diff
@@ -1,10 +1,12 @@
 {% load staticfiles %}
 {% load template_extras %}
+{% load raven %}
```

Comment:
Could we add a comment here to help soothe my bafflement regarding what `raven` and `sentry_public_dsn` are doing here?  e.g. a link to the docs.  Either as a comment or in the commit message.